contract/AElf.Contracts.Consensus.AEDPoS/AEDPoSContract_ViewMethods.cs

```diff
@@ -31,6 +31,11 @@ public partial class AEDPoSContract
                 }
                 : new MinerList();
 
+        public override PubkeyList GetCurrentMinerPubkeyList(Empty input) => new PubkeyList
+        {
+            Pubkeys = {GetCurrentMinerList(input).Pubkeys.Select(p => p.ToHex())}
+        };
+
```

Comment:
Why do we need another method that returns current miners' public keys as string? Now we have two methods (`GetCurrentMinerList` and `GetCurrentMinerPubkeyList`) with different names that just return the same thing in different format but it is not obvious from methods' names.

Reply:
> Why do we need another method that returns current miners' public keys as string? Now we have two methods (`GetCurrentMinerList` and `GetCurrentMinerPubkeyList`) with different names that just return the same thing in different format but it is not obvious from methods' names.

These are demands from our front devs. I'll discuss with them.